storm-server/src/main/java/org/apache/storm/scheduler/Cluster.java

```diff
@@ -749,12 +792,16 @@ private double calculateSharedOffHeapNodeMemory(
      */
     public void freeSlot(WorkerSlot slot) {
         // remove the slot from the existing assignments
+        String nodeId = slot.getNodeId();
```

Comment:
Since this is added, Line 808 can be updated to use `nodeId` instead of `slot.getNodeId()`

Reply:
changed